rts/Sim/Projectiles/Projectile.cpp

```diff
@@ -212,7 +212,7 @@ bool CProjectile::GetMemberInfo(SExpGenSpawnableMemberInfo& memberInfo)
 	if (CExpGenSpawnable::GetMemberInfo(memberInfo))
 		return true;
 
-	CHECK_MEMBER_INFO_FLOAT3(CProjectile, dir)
+	//CHECK_MEMBER_INFO_FLOAT3(CProjectile, dir)
```

Comment:
I don't know spring good enough to find the best way to solve this. Please let me know how you would do it.